package.json

```diff
@@ -97,6 +103,7 @@
     "lodash": "^4.17.20",
     "merge": "^1.2.1",
     "mime-db": "^1.40.0",
+    "mongoose": "^6.2.6",
```

Comment:
remove mongoose from deps or if it is required, move it to dev dependencies

Reply:
fixed

Reply:
(removed)